docs/extend.rst

```diff
@@ -4,56 +4,20 @@ Extending Chassis
 Chassis includes a flexible extension system to allow setting up the server
 exactly how you like.
 
-Available Extensions
---------------------
+Install extensions from the `Extensions Index`_
 
-This is a non-exhaustive list of what's available to drop into your Chassis box.
+.. _Extensions Index: http://beta.chassis.io/extensions/
 
-* `Memcache`_: Install a memcached server, and configure WordPress to use it
-  automatically.
 
-* `Redis`_: Install Redis server.
-
-* `phpMyAdmin`_: Set up phpMyAdmin on your Chassis box
-
-* `Tester`_: Install and configure PHPUnit, along with the WordPress additions
-  so you can run plugin unit tests.
-
-* `Theme Review`_: Set Chassis up for theme reviewing.
-
-* `MailHog`_: A Chassis extension to install and configure MailHog on your server.
-
-* `Cavalcade`_: Cavalcade is a replacement for WordPress' built-in cron that runs as a daemon on your system. It horizontally scales in production to ensure your scheduled tasks keep up with the scale of your site.
-
-* `XDebug`_: A Chassis extension to install and configure Xdebug on your server.
-
-* `Query Monitor`_: A Chassis extension to install and configure your Chassis WP install to use Query Monitor to help with debugging.
-
-* `Debugging`_: A Chassis extension to install and activate common WordPress plugins used for debugging during development.
-
-* `NodeJS`_: A Chassis extension to install NodeJS.
-
-* `Imagick`_: A Chassis extension to install ImageMagick.
+Example: Installing The Memcached Extension
+-------------------------------------------
 
-* `Fish`_: A Chassis extension to install Fish Shell.
+To install the memcached extension you would do the following:
```

Comment:
I reckon it's worth either mentioning how to add extensions in the yaml files or linking to our instructions about how to do that as well as having the cloning method.